pkg/plugin/manager.go

```diff
@@ -130,7 +131,14 @@ func (m *Manager) loadPluginFromPackage(plugin *Plugin) error {
 	}
 
 	// We transfer the information we have from the package to the plugin.
-	plugin.Name = pkg.Name
+	// Packages called `dcos-*-cli` get a simpler name as it becomes redundant
+	// in the plugin context. Eg. `dcos-enterprise-cli` -> `enterprise`.
+	matches := regexp.MustCompile("^dcos-(.+)-cli$").FindStringSubmatch(pkg.Name)
```

Comment:
While it is redundant, it's also the name of the package that was installed so I would support keeping it. Are there any other known subcommands that do that?